soccer/gameplay/plays/restarts/kick_penalty.py

```diff
@@ -9,6 +9,8 @@
 # one robot kicks the ball, the others just line up and wait
 class KickPenalty(play.Play):
 
+    in_penalty = False
```

Comment:
I'm a little confused by the logic with this (below ~ line 36).  It seems like right now, at some point the `in_penalty` variable will get set to True, then never get set back to false.  Is this trying to fix a problem with the penalty kick aborting once we leave the `setup` state?  Maybe we add something to the GameState object to handle this.